code/datums/uplink_item.dm

```diff
@@ -1070,15 +1051,15 @@ var/list/uplink_items = list()
 	name = "Energy Shield"
 	desc = "An incredibly useful personal shield projector, capable of reflecting energy projectiles and defending against other attacks."
 	item = /obj/item/weapon/shield/deployable/energy
-	cost = 16
+	cost = 10
 	gamemodes = list(/datum/game_mode/nuclear,/datum/game_mode/gang)
 	surplus = 20
 
 /datum/uplink_item/device_tools/medgun
 	name = "Medbeam Gun"
 	desc = "Medical Beam Gun, useful in prolonged firefights."
 	item = /obj/item/weapon/gun/medbeam
-	cost = 13
+	cost = 10
```

Comment:
I think the medical beam gun is a good pick up anyway and still worth the 13 TC investment. I think reducing it further is a bit mad.